### PR TITLE
Removed key from finalize arguments, because it is never used

### PR DESCRIPTION
**Pull request by lightballer**
- [x] tests and linter show no problems (`npm t`)
- [ ] tests are added/updated for bug fixes and new features
- [x] code is properly formatted (`npm run fmt`)

@tshemsedinov , check please

### CODE REVIEW

**Review by rodion-arr — COMMENTED** [review]
Looks like breaking change. Is it expected?

**Comment by lightballer** [comment]
> Looks like breaking change. Is it expected?

Finalize always calls with only 2 arguments, btw tests are passed

File: do.js [hunk 1]
```diff
@@ -102,15 +102,15 @@ Collector.prototype.done = function (callback) {
...
+Collector.prototype.finalize = function (err, data) {
   if (this.finished) return this;
   if (this.finish) {
     if (this.timer) {
       clearTimeout(this.timer);
       this.timer = null;
     }
     this.finished = true;
-    if (this.finish) this.finish(key, err, data);
+    if (this.finish) this.finish(err, data);
```
Comment (tshemsedinov) [inline do.js:113]: Finish certainly should have (err, data) contract

File: do.js [hunk 2]
```diff
@@ -102,15 +102,15 @@ Collector.prototype.done = function (callback) {
   return this;
 };
 
-Collector.prototype.finalize = function (key, err, data) {
```
Comment (tshemsedinov) [inline do.js:105]: Looks like bug, we always use (err, data) contract for finalize

**Review by tshemsedinov — APPROVED** [review]
LGTM

**Comment by tshemsedinov** [comment]
FYI: New `Collector` implementation moved here: https://github.com/metarhia/metautil/blob/master/lib/collector.js

**Pull request merged**